src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -851,7 +858,10 @@ public Task GetOngoingTaskInfo()
                     {
                         case OngoingTaskType.Replication:
 
-                            var watcher = record.ExternalReplications.Find(x => x.TaskId == key);
+                            var watcher = name != null ?
```

Comment:
I noticed that we have a "taskName" query param for subscriptions, we should probably use the same param for all other tasks